### PR TITLE
Make Prefixes case-insensitive

### PR DESCRIPTION
**Pull request by DennieDan**
What are the changes:
* `Prefix.java`: `prefix` String is made to lowercase
* `ArgumentTokenizer#findPrefixPosition`: change the whole argsString to lowercase to compare
* UG note

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -66,6 +66,13 @@ Action     | Format, Examples
 **Settle** | `settle INDEX amount/AMOUNT`<br> e.g., `settle 1 amount/500.00`
 **Exit**   | `exit`
 
+<box type="warning" header="##### Notes">
+
+* The prefixes (e.g. `a/`, `s/` or `paid`) are case-insensitive, i.e. can use `a/`, `s/` or `pAId/` instead.
+
+* You should not use the prefixes in any other cases, for example, as a content of **ADDRESS**.
```
Comment (teamotitan) [inline docs/UserGuide.md:73]: maybe "any other cases, e.g. as content of **ADDRESS**"

**Review by teamotitan — COMMENTED** [review]
Some considerations, but can ignore if you feel that it's not really good

**Comment by DennieDan** [comment]
> Some considerations, but can ignore if you feel that it's not really good

sounds good, thanks for your suggestion

**Comment by DennieDan** [comment]
> Can add in some test case to ensure that it works? For example test for edit and add with prefixes in upper case / mixed case

Sure, I am adding now

**Review by Feng1231 — APPROVED** [review]
Thank you for the effort.

File: docs/UserGuide.md (hunk 1 above)
Comment (DennieDan) [reply docs/UserGuide.md]: Thank you so much for the suggestion! I have changed it, please review.

**Pull request merged**